plugins.json

```diff
@@ -12816,6 +12816,214 @@
w6I2FkYjViNjsgc3Ryb2tlOm5vbmU7IiBkPSJNMzIxIDI5OUwzMjIgMzAwTDMyMSAyOTl6Ii8+CjxwYXRoIHN0eWxlPSJmaWxsOiM1ODZkNmM7IHN0cm9rZTpub25lOyIgZD0iTTM0NyAyOTlMMzQ4IDMwMEwzNDcgMjk5eiIvPgo8cGF0aCBzdHlsZT0iZmlsbDojYWRiNWI2OyBzdHJva2U6bm9uZTsiIGQ9Ik0zODUgMjk5TDM4NiAzMDBMMzg1IDI5OXoiLz4KPHBhdGggc3R5bGU9ImZpbGw6IzU4NmQ2Yzsgc3Ryb2tlOm5vbmU7IiBkPSJNNDExIDI5OUw0MTIgMzAwTDQxMSAyOTlNMjE3IDMwMEwyMTggMzAxTDIxNyAzMDBNMjQ5IDMwMEwyNTAgMzAxTDI0OSAzMDBNMjc3IDMwMEwyNzggMzAxTDI3NyAzMDBNMzIxIDMwMEwzMjIgMzAxTDMyMSAzMDBNMzg1IDMwMEwzODYgMzAxTDM4NSAzMDBNMjE1IDMwMUwyMTYgMzAyTDIxNSAzMDFNMjUxIDMwMUwyNTIgMzAyTDI1MSAzMDFNMjc2IDMwMUwyNzcgMzAyTDI3NiAzMDF6Ii8+CjxwYXRoIHN0eWxlPSJmaWxsOiM4MGIzOWY7IHN0cm9rZTpub25lOyIgZD0iTTM0NiAzMDFMMzQ3IDMwMkwzNDYgMzAxeiIvPgo8cGF0aCBzdHlsZT0iZmlsbDojYWRiNWI2OyBzdHJva2U6bm9uZTsiIGQ9Ik0zODYgMzAxTDM4NyAzMDJMMzg2IDMwMXoiLz4KPHBhdGggc3R5bGU9ImZpbGw6IzgwYjM5Zjsgc3Ryb2tlOm5vbmU7IiBkPSJNNDEwIDMwMUw0MTEgMzAyTDQxMCAzMDF6Ii8+CjxwYXRoIHN0eWxlPSJmaWxsOiM1ODZkNmM7IHN0cm9rZTpub25lOyIgZD0iTTIxMyAzMDJMMjE0IDMwM0wyMTMgMzAyTTI1MyAzMDJMMjU0IDMwM0wyNTMgMzAyTTI3NSAzMDJMMjc2IDMwM0wyNzUgMzAyeiIvPgo8cGF0aCBzdHlsZT0iZmlsbDojODBiMzlmOyBzdHJva2U6bm9uZTsiIGQ9Ik0zMjIgMzAyTDMyMyAzMDNMMzIyIDMwMk0zODYgMzAyTDM4NyAzMDNMMzg2IDMwMnoiLz4KPHBhdGggc3R5bGU9ImZpbGw6IzU4NmQ2Yzsgc3Ryb2tlOm5vbmU7IiBkPSJNNDEwIDMwMkw0MTEgMzAzTDQxMCAzMDJNMjExIDMwM0wyMTIgMzA0TDIxMSAzMDNNMjU1IDMwM0wyNTYgMzA0TDI1NSAzMDNNMjc0IDMwM0wyNzUgMzA0TDI3NCAzMDN6Ii8+CjxwYXRoIHN0eWxlPSJmaWxsOiNhZGI1YjY7IHN0cm9rZTpub25lOyIgZD0iTTM0NSAzMDNMMzQ2IDMwNEwzNDUgMzAzTTQwOSAzMDNMNDEwIDMwNEw0MDkgMzAzeiIvPgo8cGF0aCBzdHlsZT0iZmlsbDojNTg2ZDZjOyBzdHJva2U6bm9uZTsiIGQ9Ik0yNTYgMzA0TDI1NyAzMDVMMjU2IDMwNE0yNzMgMzA0TDI3NCAzMDVMMjczIDMwNHoiLz4KPHBhdGggc3R5bGU9ImZpbGw6I2FkYjViNjsgc3Ryb2tlOm5vbmU7IiBkPSJNMzIzIDMwNEwzMjQgMzA1TDMyMyAzMDR6Ii8+CjxwYXRoIHN0eWxlPSJmaWxsOiM1ODZkNmM7IHN0cm9rZTpub25lOyIgZD0iTTM0NSAzMDRMMzQ2IDMwNUwzNDUgMzA0eiIvPgo8cGF0aCBzdHlsZT0iZmlsbDojYWRiNWI2OyBzdHJva2U6bm9uZTsiIGQ9Ik0zODcgMzA0TDM4OCAzMDVMMzg3IDMwNHoiLz4KPHBhdGggc3R5bGU9ImZpbGw6IzU4NmQ2Yzsgc3Ryb2tlOm5vbmU7IiBkPSJNNDA5IDMwNEw0MTAgMzA1TDQwOSAzMDRNMTk0IDMwNUwxOTUgMzA2TDE5NCAzMDVNMjA4IDMwNUwyMDkgMzA2TDIwOCAzMDVNMjcyIDMwNUwyNzMgMzA2TDI3MiAzMDVNMzIzIDMwNUwzMjQgMzA2TDMyMyAzMDVNMzg3IDMwNUwzODggMzA2TDM4NyAzMDVNNjUgMzA2TDY1IDMwN0w5MiAzMDdMNjUgMzA2TTEzOCAzMDZMMTM4IDMwN0wxNjYgMzA3TDEzOCAzMDZNMTk1IDMwNkwxOTYgMzA3TDE5NSAzMDZNMjU5IDMwNkwyNjAgMzA3TDI1OSAzMDZNMjcwIDMwNkwyNzEgMzA3TDI3MCAzMDZ6Ii8+CjxwYXRoIHN0eWxlPSJmaWxsOiNhZGI1YjY7IHN0cm9rZTpub25lOyIgZD0iTTMyNCAzMDZMMzI0IDMwN0wzNDQgMzA3TDMyNCAzMDZ6Ii8+CjxwYXRoIHN0eWxlPSJmaWxsOiM4MGIzOWY7IHN0cm9rZTpub25lOyIgZD0iTTM0NCAzMDZMMzQ1IDMwN0wzNDQgMzA2eiIvPgo8cGF0aCBzdHlsZT0iZmlsbDojYWRiNWI2OyBzdHJva2U6bm9uZTsiIGQ9Ik0zODggMzA2TDM4OCAzMDdMNDA4IDMwN0wzODggMzA2eiIvPgo8cGF0aCBzdHlsZT0iZmlsbDojNTg2ZDZjOyBzdHJva2U6bm9uZTsiIGQ9Ik00MDggMzA2TDQwOSAzMDdMNDA4IDMwNk0xOTcuNjY3IDMwNy4zMzNMMTk4LjMzMyAzMDcuNjY3TDE5Ny42NjcgMzA3LjMzM00yMDQgMzA3TDIwNSAzMDhMMjA0IDMwN00yNjEuNjY3IDMwNy4zMzNMMjYyLjMzMyAzMDcuNjY3TDI2MS42NjcgMzA3LjMzM00yNjggMzA3TDI2OSAzMDhMMjY4IDMwN3oiLz4KPC9zdmc+Cg==",
```

Comment:
This icon data is massive. Any way we can scale this down?

Reply:
The long scrolling really proves my point 🤣

Reply:
The source file (https://github.com/mattermost/mattermost-plugin-servicenow/blob/e9d9ae34f20bc52872e7ab022924b582208b4ea1/assets/icon.svg?short_path=758daa0) also contains a lot of details. Hence, does the encoding.

2/5 it's not worth spending time on reducing the size of a single entry. Caching would be the next step if the amount of data that is transferred becomes too much.